README.md

```diff
@@ -105,7 +105,15 @@ Choice must also be `<None>`.
     }
 }
 ```
-
+### Using Tyr as a Library
+
+| Api                                               | Description                                                                                                                                                                                                                                                                                                                                                       |
+|---------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| async generateProject(configs)                    | The configs json object should have the same format as the .tyrfile.  This method calls all the other API methods and does all the work to set up the project.  It will not throw an error though, if an error occurs.  The error will be written to the log.  See other API methods for greater descriptions of functionality.                                   |
+| async generateBasicNodeProject(configs, filePath) | The configs json object should have the same format as the .tyrfile. The filePath is where the project will be created.,This generates the most basic node files including the src folder, index.html, index.js, package.json, README, and node-config folder. It returns the configs object.                                                                     |
```

Comment:
Use bullets instead of list for the file. 

also, use `code formatting` for the files names